crates/dojo/world/src/remote/events_to_remote.rs

```diff
@@ -250,6 +251,17 @@ impl WorldRemote {
 
                 trace!(?e, "Owner updated.");
             }
+            WorldEvent::MetadataUpdate(e) => {
+                if e.resource == Felt::ZERO {
```

Comment:
We should use the `WORLD` constant here.

Reply:
(and if we don't have one, would you mind creating it? :))

Reply:
Yep, good idea 👍